packages/headless/src/commerce.index.ts

```diff
@@ -52,11 +52,14 @@ export * from './features/commerce/facets/date-facet/date-facet-actions-loader';
 export * from './features/commerce/facets/numeric-facet/numeric-facet-actions-loader';
 export * from './features/commerce/query-set/query-set-actions-loader';
 export * from './features/commerce/query-suggest/query-suggest-actions-loader';
-export * from './features/configuration/configuration-actions-loader';
+export * from './features/commerce/configuration/configuration-actions-loader';
```

Comment:
This is breaking, please take time to review this file (and configuration-actions.ts) carefully!